.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -112,10 +121,15 @@ pipeline {
                 }
             }
         }
+
         stage('Build OptaPlanner') {
             steps {
                 script {
-                    getOptaplannerMavenCommand().withProperty('maven.test.failure.ignore', true).skipTests(params.SKIP_TESTS).run('clean install')
+                    getOptaplannerMavenCommand()
+                            .withProperty('maven.test.failure.ignore', true)
+                            .withProperty('image')
```

Comment:
Activates a profile that builds the operator's image.

Reply:
Maybe the property name is a bit too generic?

Reply:
Perhaps `operator.image` or `operator.image.build` ?

Reply:
Either is fine with me.